tests/docker_run_tests.rs

```diff
@@ -111,7 +110,7 @@ fn simple_build(path: &str) -> String {
         path,
         Vec::new(),
         &GeneratePlanOptions {
-            pin_pkgs: true,
+            pin_pkgs: false,
```

Comment:
We should be pinning the packages in the tests. Not pinning means that they could break if the latest nix unstable channel updates to a newer version of a package that we use (has happened before). If pinning doesn't work for Swift specifically then we shouldn't pin for just that test (use `create_docker_image` directly). Any idea why it doesn't work with pinning? It is not an uncommon thing to do and ideally all providers work.

Reply:
> Any idea why it doesn't work with pinning?

After further investigation, the swift versions `5.5.x` doesn't work at all.